data/hyperOutportBlocks/hyperOutportBlock.proto

```diff
@@ -511,3 +513,23 @@ message Receipt {
 message BigInt {
   bytes bytes = 1;
 }
+
+message StateChanges {
+  repeated StateChange StateChanges = 1;
+}
+
+message StateChange {
+  string Type = 1;
```

Comment:
we should think of having enum for `Type` instead of string

Reply:
also in mx-chain-go

Reply:
modified.